include/wx/strconv.h

```diff
@@ -387,6 +387,8 @@ class WXDLLIMPEXP_BASE wxMBConvUTF8 : public wxMBConvStrictUTF8
     int m_options;
 };
 
+bool wxIsUTF8Prefix(const char *src, size_t len);
```

Comment:
Ideally we shouldn't put this private function in a public header. The trouble is that we don't seem to have any appropriate file in `include/wx/private` where we could put it and creating a new header just for a single function doesn't seem appealing... But I guess we still should do it, `include/wx/private/unicode.h` maybe?

If not, we should at the very least put it into `wxPrivate` namespace but, again, ideally we wouldn't have it in the public header at all.

Oh, and I'd also rename this one to `wxCanBeUTF8Prefix()`, but this is much less important for a private function.

Reply:
I only put it in `strconv.cpp` to use `tableUtf8Lengths[]`, so maybe we should move this table to private header? This would also allow to remove its copy from `ustring.cpp`.

Reply:
I had no idea we had a copy of it there to be honest... We definitely shouldn't have it there, but I don't think we should define this array (`tableUtf8Lengths`) in the header, we'd have to make it static and it still would be duplicated. We could declare it there, of course, and still keep it defined in `strconv.cpp` and I'd be glad if you could modify it like this, of course. TIA!